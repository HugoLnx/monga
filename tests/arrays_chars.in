void main() {
	char[] hey;
	hey = new char[5];
	hey[0] = 'A';
	hey[1] = 'B';
	hey[4] = 'E';
	hey[2] = 'C';
	hey[3] = 'D';
	printf("array1: %c\n", hey[0]);
	printf("array2: %c\n", hey[1]);
	printf("array3: %c\n", hey[2]);
	printf("array4: %c\n", hey[3]);
	printf("array5: %c\n", hey[4]);
}
