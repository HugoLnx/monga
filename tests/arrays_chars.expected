array1: A
array2: B
array3: C
array4: D
array5: E
